src/modules/BootStorage.rb

```diff
@@ -21,39 +21,39 @@
 require "storage"
 require "y2storage"
 require "bootloader/udev_mapping"
+require "bootloader/exceptions"
 
 module Yast
   class BootStorageClass < Module
     include Yast::Logger
-    using Y2Storage::Refinements::DevicegraphLists
-    using Y2Storage::Refinements::Disk
 
-    attr_accessor :mbr_disk
+    # @return [Y2Storage::Partitionable]
+    attr_reader :mbr_disk
+    # @return [Y2Storage::Partition]
+    attr_reader :boot_partition
+    # @return [Y2Storage::Partition]
+    attr_reader :root_partition
+    # @return [Y2Storage::Partition, nil]
+    attr_reader :extended_partition
 
     def main
       textdomain "bootloader"
 
       Yast.import "Arch"
       Yast.import "Mode"
 
-      # string sepresenting device name of /boot partition
-      # same as RootPartitionDevice if no separate /boot partition
-      @BootPartitionDevice = ""
-
-      # string representing device name of / partition
-      @RootPartitionDevice = ""
-
-      # string representing device name of extended partition
-      @ExtendedPartitionDevice = ""
-
```

Comment:
These explanatory comments got lost. Why are not them part of the documentation of the new attr_readers?